### DISCUSSION TITLE
Jupyter kernel not starting

### DISCUSSION THREAD

**Discussion by Ambuj-Choudha** [microsoft/vscode-jupyter#10109] [Questions and Answers]
So, I am able to run the jupyter notebook using a remote server but I am not able to understand why I can't do the same with a local server. I even checked the package list of the python environment and "jupyter" is installed in that. Kindly help with issue. Thanks.

Error Note: 
Failed to start the Kernel. 
Unable to start Kernel 'base (Python 3.8.8)' due to connection timeout. 
View Jupyter [log](command:jupyter.viewOutput) for further details.

[output.txt](https://github.com/microsoft/vscode-jupyter/files/8764025/output.txt)

**Comment by IanMatthewHuff** [comment]
@Ambuj-Choudha Could you try the following?

1. Open up a command prompt and activate that base Conda environment
2. Try running `python -m ipykernel_launcher`
3. See if that command is able to startup correctly from the command line (note you might have to close the terminal to quit that process on windows)

**Reply by Ambuj-Choudha** [reply]
Thanks Ian for the reply.
The command was able to startup correctly from the anaconda prompt in that base environment.

**Reply by arogozhnikov** [reply]
Had similar problem, in my case had to change path to python interpreter

**Reply by Ambuj-Choudha** [reply]
thanks
